src/web/src/components/AboutFooter.tsx

```diff
@@ -1,56 +1,58 @@
 import { makeStyles, useTheme } from '@material-ui/core/styles';
-import { Box, Divider, Grid, Typography, useMediaQuery } from '@material-ui/core';
+import { Box, createStyles, Divider, Grid, Typography, useMediaQuery } from '@material-ui/core';
 import GitHubIcon from '@material-ui/icons/GitHub';
 import GitHubContributorCard from './GitHubContributorCard';
 import Logo from './Logo';
 import { FaSlack } from 'react-icons/fa';
 
-const useStyles = makeStyles((theme) => ({
-  root: {
-    flexGrow: 1,
-    backgroundColor: theme.palette.primary.main,
-    padding: '2rem',
-  },
-  heading: {
-    color: theme.palette.primary.contrastText,
-    [theme.breakpoints.down('md')]: {
-      marginTop: '2rem',
+const useStyles = makeStyles((theme) =>
+  createStyles({
```

Comment:
Quick question: since the problem is the MUI version, do we need to use `createStyles` here?

Reply:
@manekenpix 
Follow the [docs](https://material-ui.com/guides/typescript/#using-createstyles-to-defeat-type-widening).
Please let me know what do you think.